package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "ngx-logger",
-  "version": "2.0.0",
+  "version": "1.1.3",
```

Comment:
Our 1.* versions support angular 4, These changes are being made to master, which is now supporting Angular5 and so this version should be 2.0.1.